day_two_guide/topics/host_health.adoc

```diff
@@ -0,0 +1,111 @@
+////
+Host health
+
+Module included in the following assemblies:
+
+* day_two_guide/environment_health_checks.adoc
+////
+
+To verify that the cluster is up and running, connect to a master instance, and run the following:
+
+----
+$ oc get nodes
+NAME                  STATUS                     AGE       VERSION
+ocp-infra-node-1clj   Ready                      1h        v1.6.1+5115d708d7
+ocp-infra-node-86qr   Ready                      1h        v1.6.1+5115d708d7
+ocp-infra-node-g8qw   Ready                      1h        v1.6.1+5115d708d7
+ocp-master-94zd       Ready,SchedulingDisabled   1h        v1.6.1+5115d708d7
+ocp-master-gjkm       Ready,SchedulingDisabled   1h        v1.6.1+5115d708d7
+ocp-master-wc8w       Ready,SchedulingDisabled   1h        v1.6.1+5115d708d7
+ocp-node-c5dg         Ready                      1h        v1.6.1+5115d708d7
+ocp-node-ghxn         Ready                      1h        v1.6.1+5115d708d7
+ocp-node-w135         Ready                      1h        v1.6.1+5115d708d7
+----
+
+The above cluster example consists of three master hosts, three infrastructure
+node hosts, and three node hosts. All of them are running. All hosts in the
+cluster should be visible in this output.
+
+The `Ready` status means that master hosts can communicate with node hosts and
+that the nodes are ready to run pods (excluding the nodes in which scheduling is
+disabled).
+
+A basic `etcd` health status can be checked with the `etcdctl2` command from any
+master instance:
+
+----
+# etcdctl2 cluster-health
```

Comment:
this also work for etcd version 3?

Reply:
Fresh installed 3.7 (so, etcdv3 schema and etcd version 3):
```
[root@master-0 ~]# etcdctl2 -v
etcdctl version: 3.2.9
API version: 2
[root@master-0 ~]# etcdctl2 cluster-health
member 23510f01d64e9b29 is healthy: got healthy result from https://172.16.34.5:2379
member 2c52c55dcabc8010 is healthy: got healthy result from https://172.16.34.7:2379
member ac10cf6d7bfb2533 is healthy: got healthy result from https://172.16.34.6:2379
cluster is healthy
[root@master-0 ~]# oc version
oc v3.7.14
kubernetes v1.7.6+a08f5eeb62
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://ocp.w0t.lan:8443
openshift v3.7.14
kubernetes v1.7.6+a08f5eeb62
```